e2e-tests/release_images

```diff
@@ -5,3 +5,12 @@ IMAGE_MONGOD50=percona/percona-server-mongodb:5.0.29-25
 IMAGE_BACKUP=percona/percona-backup-mongodb:2.6.0
 IMAGE_PMM_CLIENT=percona/pmm-client:2.43.2
 IMAGE_PMM_SERVER=percona/pmm-server:2.43.2
+GKE_MIN=1.28
+GKE_MAX=1.30
+EKS_MIN=1.28
+EKS_MAX=1.31
+AKS_MIN=1.28
+AKS_MAX=1.28
+OPENSHIFT_MIN=4.13.52
+OPENSHIFT_MAX=4.17.3
+MINIKUBE=v1.31.0
```

Comment:
why minikube has `v`?

Reply:
because that's the format it expects the value to be in

Reply:
OK, I trimmed the `v` from the Minikube like, e.g. `v1.31.0` and then will add it from within the pipeline itself.